signac_dashboard/templates/layout.html

```diff
@@ -16,7 +16,7 @@
         <meta name="viewport" content="width=device-width">
         <meta name="viewport" content="initial-scale=1.0">
         <title>
-            {% block title %}Page Title{% endblock %}{% if self.subtitle() %} - {% block subtitle %}{% endblock %}{% else %} - {{ APP_NAME }}{% endif %}
+            {% block title %}Page Title{% endblock %}{% if self.subtitle() %} - {% block subtitle %}{% endblock %}{% else %}{{ APP_NAME }}{% endif %}
```

Comment:
From what I can tell, the `<title>` strings for pages without a subtitle would now look like:
```
Page Titlesignac-dashboard
```
Is that right? I think we could simplify the logic and write:
```suggestion
            {% block title %}Page Title{% endblock %} - {% if self.subtitle() %}{% block subtitle %}{% endblock %}{% else %}{{ APP_NAME }}{% endif %}
```

Reply:
I also want it to work for a page without a title, but I will look into this

Reply:
Added another if for the page title